bqplot/_version.py

```diff
@@ -1,8 +1,8 @@
-version_info = (0, 11, 8, 'final', 0)
+version_info = (0, 11, 9, 'final', 0)
```

Comment:
Maybe this commit was pushed by accident?

Reply:
yes, there is an issue with the 0.11.x branch. It is missing the commit for 0.11.9, while the tag is there.

Thankfully, there has been only two commits since. We can probably fix history.

Reply:
Oh sorry I did not realize it was 0.11